aws-dual-stack/openshift-delete-cluster.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -Eueo pipefail
+scriptName=$(basename "$0")
+
+usage() {
+  cat <<EOF_USAGE
+Created by Aspen Mesh:
+Delete any dual stack resources that block the openshift cli, uses the metadata.json so no parameters are needed
+Requires: openshift (cluster, installer, client, pull secret), aws cli and creds
+Tested on OSX 11.6
+
+Usage: ${scriptName}
+EOF_USAGE
+  exit 1
+}
+
+## Global vars
+INSTALL_DIR="_install_"
+export KUBECONFIG="$INSTALL_DIR/auth/kubeconfig"
+export OC_INSTALL_CMD="$INSTALL_DIR/openshift-install"
+export KC_CMD="$INSTALL_DIR/kubectl"
```

Comment:
Do we want to do an unset when you are done?

Reply:
Ill output a warning. the user would want to run oc get nodes and things after running the scripts